scripts/base.sh

```
@@ -33,7 +33,7 @@ repo_url="$(get_repo)"
 
 ### Print settings f this is first run
 if [ $is_first == "true" ]; then 
```

Shellcheck (0.10.0)

[warning] is_first is referenced but not assigned.

(SC2154)